gobblin-data-management/src/main/java/gobblin/data/management/retention/version/finder/PathDateTimeDatasetVersionFinder.java

```diff
@@ -27,19 +27,20 @@
 
 
 /**
- * {@link gobblin.data.management.retention.version.finder.DatasetVersionFinder} for timestamped datasets. Uses a datetime pattern
- * to find dataset versions and parse the {@link org.joda.time.DateTime} representing the version.
+ * {@link gobblin.data.management.retention.version.finder.DatasetVersionFinder} for datasets based on path timestamps.
+ * Uses a datetime pattern to find dataset versions from the dataset path
+ * and parse the {@link org.joda.time.DateTime} representing the version.
  */
-public class DateTimeDatasetVersionFinder extends DatasetVersionFinder<TimestampedDatasetVersion> {
+public class PathDateTimeDatasetVersionFinder extends DatasetVersionFinder<TimestampedDatasetVersion> {
```

Comment:
This name is certainly better, but this class might be already used in some running projects. Can we revert it to the original name? Thanks.